llama_stack/distribution/build_venv.sh

```diff
@@ -73,11 +73,16 @@ run() {
   local env_name="$1"
   local pip_dependencies="$2"
   local special_pip_deps="$3"
+  
+  if [ -n "${UV_SYSTEM_PYTHON:-}" ]; then 
```

Comment:
oh I missed this, but I think you need to initialize this before the `set -euo pipefail` otherwise you will get a failure if this env var is not set.